src/infra/marketplace/marketplaceCmdRepo.go

```diff
@@ -437,6 +429,139 @@ func (repo *MarketplaceCmdRepo) InstallItem(
 	)
 }
 
+func (repo *MarketplaceCmdRepo) uninstallSymlinkFilesRemoval(
+	installedItem entity.MarketplaceInstalledItem,
+	catalogItem entity.MarketplaceCatalogItem,
+	trashDirPath string,
+) error {
+	publicDirBackupPath := "/app/html-backup"
+	err := infraHelper.MakeDir(publicDirBackupPath)
+	if err != nil {
+		return errors.New("CreatePublicDirectoryBackupError: " + err.Error())
+	}
+
+	fileNameFilterParams := "-name \"" + catalogItem.UninstallFileNames[0].String() + "\""
+	for _, fileToIgnore := range catalogItem.UninstallFileNames[1:] {
+		fileNameFilterParams += " -o -name \"" + fileToIgnore.String() + "\""
+	}
+
+	moveFilesToKeepCmd := fmt.Sprintf(
+		"find %s/ -mindepth 1 -maxdepth 1 -not \\( %s \\) -exec mv -t %s {} +",
+		installedItem.InstallDirectory.String(),
+		fileNameFilterParams,
+		publicDirBackupPath,
+	)
+	_, err = infraHelper.RunCmdWithSubShell(moveFilesToKeepCmd)
+	if err != nil {
+		return errors.New("MoveFilesToKeepToBackupDirectoryError: " + err.Error())
+	}
+
+	srcInstallDirPath := fmt.Sprintf(
+		"/app/%s-%s-%s",
+		installedItem.AppSlug.String(),
+		installedItem.Hostname.String(),
+		installedItem.InstallUuid.String(),
+	)
+	_, err = infraHelper.RunCmdWithSubShell(
+		"mv " + srcInstallDirPath + "/* " + trashDirPath,
+	)
+	if err != nil {
+		return errors.New("MoveFilesToRemoveToTrashError: " + err.Error())
+	}
+
+	_, err = infraHelper.RunCmdWithSubShell(
+		"rm -rf " + srcInstallDirPath,
+	)
+	if err != nil {
+		return errors.New("RemoveSourceInstallDirectoryError: " + err.Error())
+	}
+
+	_, err = infraHelper.RunCmdWithSubShell(
+		"rm -rf " + infraData.GlobalConfigs.PrimaryPublicDir,
+	)
+	if err != nil {
+		return errors.New("RemovePublicDirectoryError: " + err.Error())
+	}
```

Comment:
You are absolutely correct. I changed the paths to the installation directory instead of just pointing to `/app/html`. This already solves the problem.